files/ACCA/ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc

```diff
@@ -0,0 +1,38 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition [Alignment-basedReferenceView]'), '2;1');
+FILE_NAME ('ACCA_IfcTunnel_Sprint_1_2_Georef_A.ifc', '2023-02-16T14:56:46', ('Alfonso D\X2\0027\X0\Urso', 'alfonso.durso@acca.it'), ('ACCA software S.p.A.'), 'ACCA_IFC.dll Version 500', 'ACCA usBIM.infra', $);
+FILE_SCHEMA (('IFC4X4'));

```

Comment:
We encourage you to use schema identifier with a hash attached, see [FAQ](https://github.com/bSI-InfraRoom/IFC-Tunnel-Deployment/blob/main/FAQ.md). (Not necessary to correct now, but the checker will be updated to reject files with wrong schema identifier.)

Reply:
Will try to automatize this for next sprint